src/services/database/registration-request-repository.js

```diff
@@ -7,16 +7,17 @@ import { Op } from "sequelize";
 
 const RegistrationRequest = ModelFactory.getByName(modelName);
 
-export const getRegistrationRequestStatusByConversationId = conversationId => {
-  return RegistrationRequest.findByPk(conversationId);
+export const getRegistrationRequestByConversationId = async conversationId => {
+  return await RegistrationRequest.findByPk(conversationId);
 };
 
 export const getNhsNumberByConversationId = conversationId => {
```

Comment:
Again, let's discuss the async

Reply:
To resolve on a separate ticket @AndyFlintAnswerDigital